buildSrc/build.gradle.kts

```diff
@@ -57,7 +56,9 @@ dependencies {
     implementation("org.apache.maven:maven-aether-provider:3.1.0")
 
     implementation("com.google.code.gson:gson:2.8.9")
-    implementation("com.android.tools.build:gradle:3.4.3")
+    implementation("org.jetbrains.dokka:dokka-android-gradle-plugin:0.9.17-g013-9b8280a")
```

Comment:
Do we still need this now that dokka was removed?

Reply:
Great catch, I forgot to remove as part of the merge, removed